docs/README.md

```diff
@@ -35,16 +35,5 @@ and monitor the security and cost of Kubernetes workloads.
 Learn more about
 [installing Fairwinds Insights](/installation/getting-started),
 features like [Slack notifications](/integrations/slack),
-and our supported plugins below:
-
-* [Fairwinds Polaris](https://github.com/FairwindsOps/polaris) - checks best practices in workload configuration
-* [Fairwinds Goldilocks](https://github.com/FairwindsOps/goldilocks) - recommends resource limits and requests based on actual resource usage
-* [Trivy](https://github.com/aquasecurity/trivy) - scans Docker images for known vulnerabilities
-* [OPA](https://www.openpolicyagent.org/) - specify custom checks for Kubernetes resources
-* [Kubesec](https://github.com/controlplaneio/kubesec) - checks workload configuration for security issues
-* [Kube-hunter](https://github.com/aquasecurity/kube-hunter) - checks for cluster- and node-level security vulnerabilities
-* [kube-bench](https://github.com/aquasecurity/kube-bench) - checks the cluster against the CIS Benchmark
-* Release Watcher - checks for updates to Helm 3 charts (exclusive to Fairwinds Insights)
-* [RBAC Reporter](https://github.com/FairwindsOps/insights-plugins/tree/master/rbac-reporter) - uploads metadata about RBAC profiles (exclusive to Fairwinds Insights)
-* [Workload Reporter](https://github.com/FairwindsOps/insights-plugins/tree/master/workload) - uploads high-level metadata about running workloads (exclusive to Fairwinds Insights)
+and our [supported plugins](/reports).
```

Comment:
We should probably add in the links back to the Open Source projects in the individual reports if we're ditching them here.

Reply:
They should have it already - I'll take a look through in my docs dive on Tues to make sure they all do.